### ISSUE TITLE
Add support for F#

### ISSUE THREAD

**Issue by milbrandt** [kaspersorensen/dotnet-maven-plugin#29]
Version 0.24, new fsproj build files

Seems like neither the solution file `.sln` nor the `.fsproj` files are inspected. Error reported:

> [ERROR] Failed to execute goal org.eobjects.build:dotnet-maven-plugin:0.24:restore (default-restore) on project FsSonarRunner: Could not find any directories with a project/build file. -> [Help 1]

**Comment by kaspersorensen** [comment]
That's right. Today the plugin only looks for either `project.json` files or `*.csproj` files. I don't know much about how F# projects work, but it would be a cool addition to identify those.

**Comment by milbrandt** [comment]
They are not that much from `csproj` files. There are tons of examples here on Github. I stumpled accross this plugin while upgrading here https://github.com/jmecsoftware/sonar-fsharp-plugin/blob/master/FsSonarRunner/FsSonarRunnerCore/FsSonarRunnerCore.fsproj
At the moment, it's the old style (commit 006fd6a6d9b3dad73f7026300c3133619bac3fcc) but the next commit to master will upgrade the project file to the new style.
